docs/Code_Quality_Guidelines.rst

```diff
@@ -0,0 +1,68 @@
+.. _Code_Quality_Guidelines:
+
+*******************
+TARDIS Coding Guide
+*******************
+
+General Code Format
+===================
+
+TARDIS follows the `Black <https://black.readthedocs.io/en/stable/>`_ coding style for files, and files submitted should follow the Black style. To install Black, do::
```

Comment:
Black is actually a code formatter not style guide. Style guide that we follow is PEP8.